proto/src/main/proto/Rss.proto

```diff
@@ -605,12 +605,13 @@ message RssReassignFaultyShuffleServerRequest{
   int32 shuffleId  = 1;
   repeated int32 partitionIds = 2;
   string faultyShuffleServerId = 3;
+  repeated int32 needLoadBalancePartitionIds = 4;
```

Comment:
Could you extra a more proper interface?
ReasignShuffleServer may be more proper?
Why do we need partitionIds and needLoadBalancePartitionIds?
Could we unify them into `reassignPartitions`?

Reply:
ReassignInactiveServerForPartitions {
...
map<int32, string> inactivePartitionServers;
}

ReassignServerForLoadBalance {
...
repeated int32 loadBalancePartitionIds = 4;
}

Reply:
Although the difference is not big. the sepration looks clear. I will refactor when I have time.